templates/var/lib/ansible/playbooks/ipfailover.yml

```diff
@@ -0,0 +1,9 @@
+---
+{{=<% %>=}}
+- hosts: masters[0]
+  sudo: yes
+  tasks:
+  - name: Deploy Openshift IP failover for router
+    command: oadm ipfailover --create --service-account=router --interface=eth0 --selector='{{ openshift_router_selector }}' --replicas={{ num_infra }} --virtual-ips="{{ router_vip }}" --credentials=/etc/origin/master/openshift-router.kubeconfig
```

Comment:
openshift-ansible is failing on this for me:
GATHERING FACTS *************************************************************** 
<ipf-openshift-master-0.example.com> ESTABLISH CONNECTION FOR USER: cloud-user
<ipf-openshift-master-0.example.com> REMOTE_MODULE setup
<ipf-openshift-master-0.example.com> EXEC ssh -C -tt -vvv -o ControlMaster=auto -o ControlPersist=60s -o ControlPath="/root/.ansible/cp/ansible-ssh-%h-%p-%r" -o StrictHostKeyChecking=no -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=cloud-user -o ConnectTimeout=10 ipf-openshift-master-0.example.com /bin/sh -c 'mkdir -p $HOME/.ansible/tmp/ansible-tmp-1466506793.16-199816308477047 && echo $HOME/.ansible/tmp/ansible-tmp-1466506793.16-199816308477047'
<ipf-openshift-master-0.example.com> PUT /tmp/tmpywy_m5 TO /home/cloud-user/.ansible/tmp/ansible-tmp-1466506793.16-199816308477047/setup
<ipf-openshift-master-0.example.com> EXEC ssh -C -tt -vvv -o ControlMaster=auto -o ControlPersist=60s -o ControlPath="/root/.ansible/cp/ansible-ssh-%h-%p-%r" -o StrictHostKeyChecking=no -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=cloud-user -o ConnectTimeout=10 ipf-openshift-master-0.example.com /bin/sh -c 'sudo -k && sudo -H -S -p "[sudo via ansible, key=lybvoommngfsitjrmvukiuauaoknxkbs] password: " -u root /bin/sh -c '"'"'echo BECOME-SUCCESS-lybvoommngfsitjrmvukiuauaoknxkbs; LANG=C LC_CTYPE=C /usr/bin/python /home/cloud-user/.ansible/tmp/ansible-tmp-1466506793.16-199816308477047/setup; rm -rf /home/cloud-user/.ansible/tmp/ansible-tmp-1466506793.16-199816308477047/ >/dev/null 2>&1'"'"''
ok: [ipf-openshift-master-0.example.com]

TASK: [Deploy Openshift IP failover for router] ******************************* 
<ipf-openshift-master-0.example.com> ESTABLISH CONNECTION FOR USER: cloud-user
fatal: [ipf-openshift-master-0.example.com] => One or more undefined variables: 'openshift_router_selector' is undefined

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/root/main.retry

ipf-openshift-master-0.example.com : ok=515  changed=122  unreachable=1    failed=0   
ipf-openshift-master-1.example.com : ok=308  changed=72   unreachable=0    failed=0   
ipf-openshift-master-2.example.com : ok=308  changed=72   unreachable=0    failed=0   
ipf-openshift-node-m1y0p193.example.com : ok=129  changed=28   unreachable=0    failed=0   
localhost                  : ok=46   changed=6    unreachable=0    failed=0